components/features/MentorRanking/MentorRanking.js

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+
+import {Button, Menu} from '@mui/material';
+
+import styles from '../../../utils/styles/NavLayoutStyles/SideBar.module.css';
+import topRankings from '../../../utils/constants/exData';
+import MentorItem from './MentorItem.js';
+
+export default function Leaderboard() {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div className={styles.leaderboardButton}>
+      <Button
+        className={styles.leaderboardButtonText}
+        id="basic-button"
```

Comment:
Not a good id name. Ids should only be in one spot app wide and have a descriptive name. If this requires an id give a more descriptive name or switch to className and give descriptive className for it

Reply:
Not needed, I will also remove the Aria tag pertaining to this

Reply:
@TristenUrban I think you can import the Main Button here and use that instead